website/content/docs/ecs/architecture.mdx

```diff
@@ -48,13 +60,13 @@ One ECS task in the cluster contains the controller container, which performs th
 
 Deploying Consul to ECS starts the following process to build the architecture:
 
-1. The control-plane container starts and logs into Consul.
-1. The control-plane container registers services and proxies with the Consul servers.
-1. The control-plane container writes the bootstrap configuration for the Consul dataplane process and stores it in a shared volume.
-1. The dataplane container starts and configures itself using the bootstrap configuration generated by the control-plane container. 
-1. The dataplane container starts the Envoy sidecar proxy.
-1. The control-plane container starts listening for ECS health checks.
-1. When the ECS task indicates that the application instance is healthy, the control-plane container marks the service as healthy and starts allowing traffic to flow.
+1. The `mesh-init` container starts and logs into Consul.
+1. The `mesh-init` container registers services and proxies with the Consul servers.
+1. The `mesh-init` container writes the bootstrap configuration for the Consul dataplane process and stores it in a shared volume.
```

Comment:
Can we add a line here saying that `mesh-init` configures Consul DNS and modifies traffic redirection rules as well?